app/src/main/java/app/revanced/integrations/tiktok/settings/preference/categories/DownloadsPreferenceCategory.java

```diff
@@ -7,7 +7,7 @@
 import app.revanced.integrations.tiktok.settings.preference.DownloadPathPreference;
 import app.revanced.integrations.tiktok.settings.preference.TogglePreference;
 
-@SuppressWarnings("deprecation")
+@SuppressWarnings({"deprecation", "unused"})
```

Comment:
While I understand this change, it's annoying to work with when there's unused code. Just want to let you know.